### PR TITLE
DEV-256: Add build tags to Windows launcher

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20783/HighFidelity-Beta-PR16038-fd24889.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20783/HighFidelity-Beta-Interface-PR16038-fd24889.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20783/HighFidelity-Beta-PR16038-fd24889.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20783/HighFidelity-Beta-Interface-PR16038-fd24889.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20783/HighFidelity-Beta-PR16038-fd24889.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20783/HighFidelity-Quest-Beta-PR16038-fd24889.apk)

File: launchers/win32/LauncherManager.cpp [hunk 1]
```diff
@@ -665,7 +701,7 @@ BOOL LauncherManager::installContent() {
         }
         else {
             addToLog(_T("Error decompressing content zip file."));
-            _hasFailed = true;
+            setFailed(_hasFailed);
```
Comment (luiscuenca) [inline launchers/win32/LauncherManager.cpp]: Did you mean `setFailed(true)`?

File: launchers/win32/LauncherManager.cpp [hunk 2]
```diff
@@ -705,7 +741,7 @@ BOOL LauncherManager::downloadFile(ProcessType type, const CString& url, CString
                 } else {
                     addToLog(_T("Error downloading application."));
                 }
-                _hasFailed = true;
+                setFailed(_hasFailed);
```
Comment (luiscuenca) [inline launchers/win32/LauncherManager.cpp]: Did you mean `setFailed(true)`?

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20836/HighFidelity-Beta-PR16038-2eac3cd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20836/HighFidelity-Beta-Interface-PR16038-2eac3cd.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20836/HighFidelity-Beta-PR16038-2eac3cd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20836/HighFidelity-Beta-Interface-PR16038-2eac3cd.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20836/HighFidelity-Beta-PR16038-2eac3cd.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20836/HighFidelity-Quest-Beta-PR16038-2eac3cd.apk)

**Comment by shanzzam** [comment]
Just putting a DNM on here so I don't forget and accidentally merge it.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20921/HighFidelity-Beta-PR16038-79fa155.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20921/HighFidelity-Quest-Beta-PR16038-79fa155.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20921/HighFidelity-Beta-PR16038-79fa155.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20921/HighFidelity-Beta-Interface-PR16038-79fa155.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20921/HighFidelity-Beta-PR16038-79fa155.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20921/HighFidelity-Beta-Interface-PR16038-79fa155.exe

**Comment by AndySeattle** [comment]
Re-added DO NOT MERGE till we get go ahead to add this to master / verify mac also.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21017/HighFidelity-Beta-PR16038-cf64492.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21017/HighFidelity-Beta-Interface-PR16038-cf64492.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21017/HighFidelity-Beta-PR16038-cf64492.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21017/HighFidelity-Beta-Interface-PR16038-cf64492.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21017/HighFidelity-Beta-PR16038-cf64492.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21017/HighFidelity-Quest-Beta-PR16038-cf64492.apk)

**Pull request merged**